docs/tutorials/getting_started/moran.rst

```diff
@@ -0,0 +1,49 @@
+Moran Process
+=============
+
+The strategies in the library can be pitted against one another in the
+[Moran process](https://en.wikipedia.org/wiki/Moran_process), a population
+process simulating natural selection. The process works as follows. Given an
+initial population of players, the population is iterated in rounds consisting
+of:
+- matched played between each pair of players, with the cumulative total
+scores recored
+- a player is chosen to reproduce proportional to the player's score in the
+round
+- a player is chosen at random to be replaced
+
+The process proceeds in rounds until the population consists of a single player
+type. That type is declared the winner. To run an instance of the process with
+the library, proceed as follows::
+
+    >>> import axelrod as axl
+    >>> players = [axl.Cooperator(), axl.Defector(),
+    ...               axl.TitForTat(), axl.Grudger()]
+    >>> mp = axl.MoranProcess(players)
+    >>> mp.play()
+    >>> mp.winner   # doctest: +SKIP
+
+    Defector
+
+You can access some attributes of the process, such as the number of rounds::
+    >>> len(mp)  # doctest: +SKIP
+    6
+
+The sequence of populations::
+    >>> import pprint
+    >>> pprint.pprint(mp.populations)  # doctest: +SKIP
+    [Counter({'Defector': 1, 'Cooperator': 1, 'Grudger': 1, 'Tit For Tat': 1}),
+    Counter({'Defector': 1, 'Cooperator': 1, 'Grudger': 1, 'Tit For Tat': 1}),
+    Counter({'Defector': 2, 'Cooperator': 1, 'Grudger': 1}),
+    Counter({'Defector': 3, 'Grudger': 1}),
+    Counter({'Defector': 3, 'Grudger': 1}),
+    Counter({'Defector': 4})]
+
+The scores in each round::
+    >>> import pprint
+    >>> pprint.pprint(mp.score_history)  # doctest: +SKIP
```

Comment:
On `further_topics/probabilistict_end_tournaments.rst` we have:

```
>>> results = tournament.play()
>>> m = results.payoff_matrix
>>> for row in m:  
...     print([round(ele, 1) for ele in row]) # Rounding output  
```

Perhaps do that for consistency?